app/controllers/Application.scala

```diff
@@ -61,9 +61,11 @@ class Application @Inject()(data: DataHelper,
       val future = actions.collect(filter, categoryArray, projectsPerPage, offset, ordering)
       val projects = this.service.await(future).get
 
+      val total = this.service.await(this.service.getActions(classOf[ProjectActions]).count()).get
```

Comment:
Also the `ProjectActions` instance is already in scope so you should just use that instead of looking it up again.